ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/linearoptimisation/algorithms/linearproblem/LinearProblem.java

```diff
@@ -81,9 +82,10 @@ public static LinearProblemBuilder create() {
         return new LinearProblemBuilder();
     }
 
-    LinearProblem(List<ProblemFiller> fillerList, RangeActionsOptimizationParameters.Solver solver, double relativeMipGap, String solverSpecificParameters) {
+    LinearProblem(List<ProblemFiller> fillerList, RangeActionActivationResult raActivationFromParentLeaf, RangeActionsOptimizationParameters.Solver solver, double relativeMipGap, String solverSpecificParameters) {
```

Comment:
what happens in the cases where the activation is reset from one leaf to the next?

Reply:
I'm not sure I understand your question ; but for me the new version is strictly equivalent to the old one (see changes in LinearProblem, LinearProblemBuilder, and fillers)